gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    savefig('gaussPlot2dDemoContour_Colorbar', tight_layout=False)
</pre><p>Don't add colorbar in figname. keep same figname as previously used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Previously there were three different figures that were created for each gaussian. Now I am creating a subplot so can't have the old name